src/hotspot/share/gc/g1/g1BlockOffsetTable.cpp

```diff
@@ -102,18 +102,18 @@ G1BlockOffsetTablePart::G1BlockOffsetTablePart(G1BlockOffsetTable* array, HeapRe
 //      Move back N (e.g., 8) entries and repeat with the
 //        value of the new entry
 //
-void G1BlockOffsetTablePart::set_remainder_to_point_to_start_incl(u_char* start_card, u_char* end_card) {
+void G1BlockOffsetTablePart::set_remainder_to_point_to_start_incl(uint8_t* start_card, uint8_t* end_card) {
   assert(start_card <= end_card, "precondition");
   assert(start_card > _bot->entry_for_addr(_hr->bottom()), "Cannot be first card");
   assert(_bot->offset_array(start_card-1) < CardTable::card_size_in_words(),
          "Offset card has an unexpected value");
-  u_char* start_card_for_region = start_card;
-  u_char offset = max_jubyte;
+  uint8_t* start_card_for_region = start_card;
+  uint8_t offset = max_jubyte;
```

Comment:
Pre-existing: That should probably be `UINT8_MAX` from stdint/cstdint.h (should already be included).

Reply:
Fixed.